### PR TITLE
Add instructions to create MOTChallenge-STEP.

### PR DESCRIPTION
**Pull request by markweberdev**
Add instructions to create MOTChallenge-STEP.

Add support to read jpg.

Add minor improvements to the KITTI-STEP tutorial.

### CODE REVIEW

File: data/data_utils.py [hunk 1]
```diff
@@ -110,7 +110,8 @@ def create_features(image_data,
 
   Args:
     image_data: String or byte stream of encoded image data.
-    image_format: String, image data format, should be either 'jpeg' or 'png'.
+    image_format: String, image data format, should be either 'jpeg', 'jpg' or
+      'png'.
```
Comment (ClaireXie) [inline data/data_utils.py:114]: 'jpeg', 'jpg', or 'png' (comma)

**Review by ClaireXie — APPROVED** [review]
Thanks Mark!

File: g3doc/setup/motchallenge_step.md [hunk 2]
```diff
@@ -31,19 +31,68 @@ void           | 255
...
+    ```bash
+    wget ${MOTCHALLENGE_LINK} 
+    unzip ${MOTCHALLENGE_IMAGES}.zip
+    ```
+
+3.  Move and rename the data:
+    ```bash
+    # Create directories.
+    mkdir train
+    mkdir train/0002
```
Comment (aquariusjay) [inline g3doc/setup/motchallenge_step.md:52]: Also
mkdir train/0009

**Review by aquariusjay — COMMENTED** [review]
Thanks, Mark!

File: data/data_utils.py (hunk 1 above)
Comment (markweberdev) [reply data/data_utils.py:114]: Done.

File: g3doc/setup/motchallenge_step.md (hunk 2 above)
Comment (markweberdev) [reply g3doc/setup/motchallenge_step.md:52]: Done

**Comment by ClaireXie** [comment]
Thanks Mark. Merging this PR.

**Pull request merged**